src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -693,6 +693,10 @@ def add_electrical_series(
         assert es_key in metadata["Ecephys"], f"metadata['Ecephys'] dictionary does not contain key '{es_key}'"
         eseries_kwargs.update(metadata["Ecephys"][es_key])
 
+    if checked_recording.get_num_segments() > 1:
+        width = int(np.ceil(np.log10(checked_recording.get_num_segments())))
+        eseries_kwargs["name"] += f"{segment_index:0{width}}"
```

Comment:
```suggestion
    # If the recording extractor has more than 1 segment, append numbers to the names so that the names are unique. 0-pad these names based on the number of segments. If there are 10 segments use 2 digits, if there are 100 segments use 3 digits, etc.
    if checked_recording.get_num_segments() > 1:
        width = int(np.ceil(np.log10(checked_recording.get_num_segments())))
        eseries_kwargs["name"] += f"{segment_index:0{width}}"
```